lib/sharing/gists.dart

```diff
@@ -189,6 +200,33 @@ $styleRef$dartRef  </head>
     }
   }
 
+  Future<Gist> loadGistFromAPIDocs(String sampleId) async {
+    final contents = await HttpRequest.getString('$_apiDocsUrl/$sampleId.dart');
```

Comment:
should probably throw a `GistLoaderException` if this 404s

Reply:
Yeah, that's clearly an oversight.

Fixed.